include/ocpp/common/types.hpp

```diff
@@ -558,6 +558,17 @@ firmware_status_notification_to_firmware_status_enum_type(const FirmwareStatusNo
 
 } // namespace conversions
 
+namespace security {
+// The security profiles defined in OCPP 2.0.1 resp. in the OCPP 1.6 security-whitepaper.
+enum SecurityProfile { // no "enum class" because values are used in implicit `switch`-comparisons to `int
+                       // security_profile`
+    UNSPECIFIED = 0,
```

Comment:
Rather:
UNSECURED_TRANSPORT_WITHOUT_BASIC_AUTHENTICATION

Reply:
I've added the prefix `OCPP_1_6_` to clearly indicate it's only possible within 0CPP1.6.